### PR TITLE
fix(api,cdn): new signatures

### PR DESCRIPTION
**Pull request by fsamin**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-cds#20604.0 ✔ 
* 
  * Build and Package All -   
  * Build and Package Contrib Action ✔ 
  * Build and Package Contrib Code Plugin ✔ 
  * Build and Package Contrib Integrations ✔ 
  * Build and Package Engine, Worker, CLI and Doc ✔ 
  * Build and Package UI ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Contrib ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report integration-cds#20604.0 ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✔ 
  * Standalone Tests v2 ✔

**Review by yesnault — APPROVED** [review]

**Review by sguiheux — APPROVED** [review]

File: engine/api/integration/gorp_model.go [hunk 1]
```diff
@@ -47,9 +47,10 @@ type dbProjectIntegration struct {
 }
 
 func (e dbProjectIntegration) Canonical() gorpmapper.CanonicalForms {
-	var _ = []interface{}{e.IntegrationModelID, e.ProjectID}
+	var _ = []interface{}{e.IntegrationModelID, e.ProjectID, e.Config}
 	return gorpmapper.CanonicalForms{
 		"{{.IntegrationModelID}}{{.ProjectID}}",
+		"{{.IntegrationModelID}}{{.ProjectID}}",
```
Comment (richardlt) [inline engine/api/integration/gorp_model.go:53]:
```suggestion
```

File: engine/gorpmapper/table_mapping.go [hunk 2]
```diff
@@ -123,12 +125,25 @@ func (m *Mapper) NewTableMapping(target interface{}, name string, autoIncrement
...
 				"print": func(i interface{}) string {
 					return fmt.Sprintf("%v", err)
 				},
 				"printDate": func(i time.Time) string {
 					return i.In(time.UTC).Format(time.RFC3339)
 				},
+				"md5sum": func(i interface{}) string {
+					var dataBts []byte
+					dataString, is := i.(string)
+					if is {
```
Comment (richardlt) [inline engine/gorpmapper/table_mapping.go:140]: !is ?

**Review by richardlt — CHANGES_REQUESTED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-cds#20613.0 ✘  
* 
  * Build and Package All -   
  * Build and Package Contrib Action ✔ 
  * Build and Package Contrib Code Plugin ✔ 
  * Build and Package Contrib Integrations ✔ 
  * Build and Package Engine, Worker, CLI and Doc ✔ 
  * Build and Package UI ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Contrib ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report build-cds#20613.1 ✔ 
* 
  * Build and Package All -   
  * Build and Package Contrib Action ✔ 
  * Build and Package Contrib Code Plugin ✔ 
  * Build and Package Contrib Integrations ✔ 
  * Build and Package Engine, Worker, CLI and Doc ✔ 
  * Build and Package UI ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Contrib ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report integration-cds#20613.1 ✘  
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✔ 
  * Standalone Tests v2 ✘

**Comment by ovh-cds** [comment]
CDS Report integration-cds#20613.2 ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✔ 
  * Standalone Tests v2 ✔

**Pull request merged**